packages/web-react/src/constants/direction.ts

```diff
@@ -2,3 +2,8 @@ export const Direction = {
   HORIZONTAL: 'horizontal',
   VERTICAL: 'vertical',
 } as const;
+
+export const DirectionShort = {
```

Comment:
DirectionShort sounds a bit weird to me. How about AxisDirection

Reply:
I chose DirectionAxis